Gopkg.toml

```diff
@@ -38,13 +38,13 @@ required = ["github.com/wadey/gocovmerge"]
 
 [[override]]
   name = "github.com/docker/cli"
-  branch = "master"
+  branch = "19.03"
 
 # Waiting for https://github.com/deislabs/duffle/pull/682 to be merged
 [[override]]
   name = "github.com/deislabs/duffle"
-  source = "github.com/ijc/duffle"
-  branch = "set-merge"
+  source = "github.com/silvin-lubecki/duffle"
```

Comment:
what is required here that is not upstream ? (the comment above is out-of-date as the PR is merged)

Reply:
Since the creation of the PR both required duffle PR (https://github.com/deislabs/duffle/pull/682 and https://github.com/deislabs/duffle/pull/704) have been merged into master. Updating vendoring to reflect that